README.md

```diff
@@ -1,14 +1,16 @@
 # GTFSRT-Best-Practices
 
-Best Practices for Structuring GTFS Realtime Data
+Best Practices for Structuring GTFS Realtime Data. The main document can be found in the [index.md file](./index.md).
```

Comment:
Should read "GTFS-Realtime" to be consistent with the rest of the document.

Reply:
And on further reflection, we're using a combination of "GTFS-Realtime" and "GTFS-realtime" throughout the document. My preference is for the latter, but it should just be consistent in any case.